bedrock/mozorg/templates/mozorg/home/home-en.html

```diff
@@ -116,56 +113,55 @@ <h3 class="primary-title-sub">{{ _('Fast for good.') }}</h3>
         )}}
+        <!-- 2 -->
         {{ card_block(
             tag_label=_('News'),
-            title=_('The New York Times reviews Firefox'),
-            ga_title='The New York Times reviews Firefox',
-            desc=_('The New York Times declares Firefox is back, and it’s time to give it a try.'),
-            image_url='home/2018/cards/nyt.png',
-            include_highres_image=True,
+            title=_('Bloomberg: Pocket’s 30M users are great for publishers'),
+            ga_title='Bloomberg: Pocket’s 30M users are great for publishers',
+            desc=_('Mozilla’s focus on leisurely reads is aimed at publishers burned by Facebook. It’s paying off: Traffic is already up 75 percent this year.'),
+            image_url='home/2018/cards/pocket-article-illustration.jpg',
             aspect_ratio='mzp-has-aspect-3-2',
-            link_url='https://www.nytimes.com/2018/06/20/technology/personaltech/firefox-chrome-browser-privacy.html'
+            link_url='https://www.bloomberg.com/news/articles/2018-08-09/pocket-s-30-million-users-are-great-for-publishers'
         )}}
+        <!-- 3 -->
         {{ card_block(
-            tag_label=_('News'),
-            title=_('The Verge: The future of news feeds'),
+            tag_label=_('Video'),
+            media_icon='mzp-has-video',
+            title=_('Our privacy philosophy explained'),
             ga_title='The Verge: The future of news feeds',
-            desc=_('Casey and Nate talk about the future of personalized recommendations and how Pocket is building a better news feed.'),
-            image_url='home/2018/cards/verge-pocket.png',
-            include_highres_image=True,
+            desc=_('These five principles guide how we put people first when we make products, manage data and advocate for policies.'),
+            image_url='home/2018/cards/privacy-philosophy.png',
             aspect_ratio='mzp-has-aspect-3-2',
-            link_url='https://www.theverge.com/2018/6/13/17446660/mozilla-firefox-pocket-recommendations-ceo-nate-weiner-interview-converge-podcast'
+            link_url='https://blog.mozilla.org/internetcitizen/2018/08/13/firefox-privacy-philosophy/',
+            youtube_id='fbF9gDSZX6U',
         )}}
+        <!-- 4 -->
         {{ card_block(
-            tag_label=_('Firefox'),
-            title=_('Get the newest Firefox'),
-            ga_title='Get the newest Firefox',
-            desc=_('The latest Firefox is here. Built to help you work smarter and faster.'),
-            image_url='home/2018/cards/firefox.png',
-            include_highres_image=True,
+            tag_label=_('Podcast'),
+            title=_('IRL: Paid attention'),
+            ga_title='IRL: Paid attention',
+            desc=_('There’s a new currency in town (and no, we’re not talking about Bitcoin). We’re talking about attention. Tune in to IRL.'),
+            image_url='home/2018/cards/irl-s3-e3.png',
             aspect_ratio='mzp-has-aspect-3-2',
```

Comment:
Should this be `mzp-has-aspect-1-1`? The source image seems square, but visually it works with either aspect ratio. If they want to keep it 3:2, could we crop the image to save some bytes?